### ISSUE TITLE
support production server URL

### ISSUE THREAD

**Issue by aroman** [AEPi-AK/barnyard-audio#2]
when not developing locally (i.e. on the barnyard LAN), the server is accessible at `http://barnyard-nuc.local`

**Comment by hwangy** [comment]
Yeah, I'm currently testing using a local version of the server, which is on my localhost
